scripts/mdbook-forc-documenter/README.md

```diff
@@ -15,16 +15,24 @@ This preprocessor is automatically run on every build, as long as the `book.toml
 Enter a new entry under the `Commands` section within `SUMMARY.md`, in this format:
 
 ```md
-- [forc gm](./forc_gm.md)
+- [forc explore](./forc_explore.md)
 ```
 
+### Adding a new forc plugin
+
+Do the same as the above, with an extra step of adding an installation step within the CI. The preprocessor needs to be aware of the plugin when building the book, since it is calling `forc <plugin> --help` to generate the documentation. You can add this installation step within [`ci.yml`](https://github.com/FuelLabs/sway/blob/master/.github/workflows/ci.yml#L126) and [`gh-pages.yml`](https://github.com/FuelLabs/sway/blob/master/.github/workflows/gh-pages.yml#L26).
```

Comment:
Make these permalinks (press `y` when you have the yml files open on GitHub to get a link to the exact commit). It's better than letting the links drift and users having no idea what it's supposed to point to. At least with permalinks they can trace updates.

Reply:
TIL